packages/devtools_test/lib/src/mocks/mocks.dart

```diff
@@ -0,0 +1,699 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Just an FYI for future: if you use your IDE to move a file instead of creating a new file and copying and pasting content into the new file, this will help with git diff. It would show up as a move with some edits instead of a file delete and a file addition.

Reply:
thx